src/main/java/org/mastodon/revised/mamut/feature/DefaultMamutFeatureComputerService.java

```diff
@@ -0,0 +1,223 @@
+package org.mastodon.revised.mamut.feature;
+
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Map;
+import java.util.Set;
+
+import org.mastodon.graph.algorithm.TopologicalSort;
+import org.mastodon.graph.object.ObjectEdge;
+import org.mastodon.graph.object.ObjectGraph;
+import org.mastodon.graph.object.ObjectVertex;
+import org.mastodon.revised.mamut.ProgressListener;
+import org.mastodon.revised.model.feature.Feature;
+import org.mastodon.revised.model.feature.FeatureComputer;
+import org.mastodon.revised.model.feature.FeatureComputerService;
+import org.mastodon.revised.model.mamut.Model;
+import org.scijava.InstantiableException;
+import org.scijava.app.StatusService;
+import org.scijava.log.LogService;
+import org.scijava.plugin.Parameter;
+import org.scijava.plugin.Plugin;
+import org.scijava.plugin.PluginInfo;
+import org.scijava.plugin.PluginService;
+import org.scijava.service.AbstractService;
+
+@Plugin( type = FeatureComputerService.class )
+public class DefaultMamutFeatureComputerService extends AbstractService implements FeatureComputerService< Model >
```

Comment:
There seems to be almost nothing MaMuT-specific about this. Pull out `Abstract(Default)ComputerService`?

Reply:
Addressed in 1064e1003c8f02bc6447c1f3275958ee1cb60ffa